tools-and-tests/suites/src/main/java/org/hiero/block/suites/BaseSuite.java

```diff
@@ -127,6 +177,25 @@ protected static GenericContainer<?> createContainer() {
         return blockNodeContainer;
     }
 
+    protected static GenericContainer<?> createContainer(BlockNodeConfig config) {
+        String blockNodeVersion = BaseSuite.getBlockNodeVersion();
+        var blockNodePort = config.getPort();
+        var blockNodeMetricsPort = config.getMetricPort();
+        List<String> portBindings = new ArrayList<>();
+        portBindings.add(String.format("%d:%2d", blockNodePort, blockNodePort));
+        portBindings.add(String.format("%d:%2d", blockNodeMetricsPort, blockNodeMetricsPort));
```

Comment:
Might be good to replace these strings with constants:
```suggestion
  private static final String PORT_BINDING_FORMAT = "%d:%2d"
        portBindings.add(PORT_BINDING_FORMAT.formatted(blockNodePort, blockNodePort));
        portBindings.add(PORT_BINDING_FORMAT.formatted(blockNodeMetricsPort, blockNodeMetricsPort));
```

Also, just curious, why do we have the 2 digit minimum width (space padded) in the second half?
for example, "%d:%2d".formatted(17635, 17635) produces "17635:17635" but "%d:%2d".formatted(5, 5) produces "5: 5", which is a bit confusing...
I wonder if this was intended to be "%d:%02d" or "%d:%05d" (or perhaps "%d:%2$d" if the intent was to select the second argument).